var/spack/repos/builtin/packages/py-bbp-morphology-workflow/package.py

```diff
@@ -0,0 +1,22 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+
+from spack import *
+
+
+class PyBbpMorphologyWorkflow(PythonPackage):
+    """Python library neuron morphology analysis"""
+
+    homepage = "https://bbpcode.epfl.ch/code/#/admin/projects/platform/bbp-morphology-workflow"
+    git      = "ssh://bbpcode.epfl.ch/platform/bbp-morphology-workflow"
+    version('develop', branch='master')
```

Comment:
Why do we need those? Is it for local spack env?

Reply:
I don't know. Should I remove it ?

Reply:
depends...during daily development we install latest master locally (for our packages) and test the stuff. If this is not the case for this package, can be removed.